core/desugarSessionExceptions.ml

```diff
@@ -130,6 +129,56 @@ let contains_session_exceptions prog =
   (o#program prog)#satisfied
 
 
+(*
```

Comment:
Looks like you are merging in @SimonJF's work.

Reply:
Damn. Wasn't paying attention while shuffling things around. I'll fix this tomorrow.

Reply:
@frank-emrich's recent PR also appears in the delta.

Reply:
Git can be a joy sometimes :upside_down_face: